install/conda.rst

```diff
@@ -25,6 +25,10 @@ If not, you can quickly get started by following Continuum's official installati
 - `Install Anaconda <https://www.continuum.io/downloads>`__ if you want a complete, science-ready Python installation with minimal setup.
 - Otherwise, `install the leaner Miniconda <http://conda.pydata.org/miniconda.html>`__ version and install packages as you need them.
 
+.. warning:: 
+
+   You need the Python 2 (eg. Miniconda2) distribution. If you want to use the experimental Python 3 support, you should try our source installation. 
```

Comment:
It's a work in progress. I don't have a current Epic for the work. So it's whatever I can do outside of work. So far so good though. I built it for 3.5 and that worked. I need to address some conda-build issues to get it to work for 3.6.